tests/utils/price.js

```diff
@@ -0,0 +1,9 @@
+import { BigNumber } from 'bignumber.js';
+
+export const impliedPrice = ({ amountPriced, amountUnpriced, price }) => {
```

Comment:
What's this func for, missed it

Reply:
just a helper function i wrote to help test the result of the tax model. i just shared it because I found myself reusing it.